### ISSUE TITLE
dbeaver won't accept secp521r1 client keys

### ISSUE THREAD

**Issue by rmoog** [serge-rider/dbeaver#1063]
DBeaver won't accept secp521r1 client keys. PostgreSQL 9.6 is fine with that though.

```
Could not read SSL key file /home/rmoog/.postgresql/domain.postgresql.rmoog.key.
  java.io.IOException: extra data given to DerValue constructor
```

The file is structured as such:
```
-----BEGIN EC PARAMETERS-----
base64, usually 1 line
-----END EC PARAMETERS-----
-----BEGIN EC PRIVATE KEY-----
base64, several lines
-----END EC PRIVATE KEY-----
```

**Comment by serge-rider** [comment]
Not sure that this will 100% solve the problem but still:
Maybe you need to patch JRE security settings (see #843). Windows version already contains patched JRE but for Linux you have to do this by yourself.

**Comment by r-richmond** [comment]
On mac `brew cask install jce-unlimited-strength-policy` solved #843 I imagine some app-get command will work for linux.
